.github/auto-assignees.yaml

```diff
@@ -13,6 +13,7 @@ reviewGroups:
     - steven-zou
     - bitsf
     - wy65701436
+    - nicolaslacroux
```

Comment:
Are you planning to review PRs frequently? If yes, then add yourself to the `active` group above.

Reply:
Absolutely, I'm planning to review PRs frequently in support of Thomas.